client/my-sites/customer-home/cards/tasks/task.jsx

```diff
@@ -117,7 +118,7 @@ const Task = ( {
 					<ActionPanelTitle>{ title }</ActionPanelTitle>
 					<p className="task__description">{ description }</p>
 					<ActionPanelCta>
-						<Button className="task__action" primary href={ actionUrl }>
+						<Button className="task__action" primary onClick={ actionOnClick } href={ actionUrl }>
```

Comment:
As a side-note allowing folks, to pass-though children here might be worthwhile, so we can reduce prop passing. (Our primary action in this case is a link styled like a button).

Reply:
I just realised that something like `target={ actionIsExternal ? '_blank' : undefined }` could be used instead of this prop, but I was assuming that the link should open in a new tab since it leads to a page outside of Calypso - is that assumption correct? :)

Reply:
No need for changes here for now 👍. Just noticing that we can use more native elements if we refactor slightly in the future.